tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/InteractiveFlowTests.cs

```diff
@@ -212,31 +212,51 @@ public async Task ValidateCcsHeadersForInteractiveAuthCodeFlowAsync()
 
             Assert.IsNotNull(authResult);
             Assert.IsNotNull(authResult.AccessToken);
+        }
+
+        [TestMethod]
+        public async Task Interactive_Arlington_MultiCloudSupport_AADAsync()
+        {
+            // Arrange
+            LabResponse labResponse = await LabUserHelper.GetArlingtonUserAsync().ConfigureAwait(false);
+            var tuple = getPCA(labResponse, false, true);
+            IPublicClientApplication pca = tuple.Item1;
+
+            Trace.WriteLine("Part 1 - Acquire a token interactively");
+            AuthenticationResult result = await pca
+                .AcquireTokenInteractive(s_scopes)
+                .WithCustomWebUi(CreateSeleniumCustomWebUI(labResponse.User, Prompt.SelectAccount, false))
+                .ExecuteAsync(new CancellationTokenSource(_interactiveAuthTimeout).Token)
+                .ConfigureAwait(false);
+
+            Assert.IsNotNull(result);
+            Assert.IsNotNull(result.Account);
+            Assert.AreEqual(labResponse.Lab.Authority + labResponse.Lab.TenantId, result.Account.GetAuthority());
+
+            Trace.WriteLine("Part 2 - Get Accounts");
+            var accounts = await pca.GetAccountsAsync().ConfigureAwait(false);
+
+            Assert.IsNotNull(accounts);
+            Assert.IsTrue(accounts.Count() > 0);
```

Comment:
Should be exactly 1. 

nit: you can use `.Single()` which will fail if there isn't exactly 1